modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/components/Sidebar/Sidebar.es.js

```diff
@@ -243,24 +243,15 @@ class Sidebar extends Component {
 	}
 
 	isActionsDisabled() {
-		const {defaultLanguageId, editingLanguageId} = this.props;
-
-		return defaultLanguageId !== editingLanguageId;
+		return false;
 	}
 
 	isChangeFieldTypeEnabled() {
 		return !this.isActionsDisabled();
 	}
 
 	isFieldReadOnly({localizable, type}) {
-		const {defaultLanguageId, editingLanguageId} = this.props;
-
-		return (
-			defaultLanguageId !== editingLanguageId && (
-				!localizable ||
-				type === 'validation'
-			)
-		);
+		return false;
```

Comment:
Hi @natocesarrego 

I added this as a temporary workaround to prevent the error message field to be disabled in translations. Making the translated fields read-only was made in [LPS-93968](https://issues.liferay.com/browse/LPS-93968), and it prevents the error message to be localized.
My workaround is admittedly really simple and stupid, but I could not think of another way at the moment to allow me to make the fields localizable.

Reply:
Thanks for your clarification, @IstvanD! 😄